pyomo/contrib/parmest/parmest.py

```diff
@@ -539,51 +1045,113 @@ def _Q_opt(
                     )
                 )
 
+                self.inv_red_hes = inv_red_hes
+            elif kwargs and all(arg.value in kwargs for arg in UnsupportedArgsLib):
+                deprecation_warning(
+                    "You're using a deprecated call to the `theta_est()` function "
+                    "with the `calc_cov` and `cov_n` arguments. This usage will be "
+                    "removed in a future release. Please update to the new parmest "
+                    "interface using `cov_est()` function for covariance calculation.",
+                    version="6.7.2",
+                )
```

Comment:
Unless this is an old deprecation warning that you are moving, please change the version to `6.9.3.dev0`.